ydb/_grpc/grpcwrapper/ydb_topic.py

```diff
@@ -410,17 +410,23 @@ def to_proto(self) -> ydb_topic_pb2.StreamReadMessage.InitRequest:
         class TopicReadSettings(IToProto):
             path: str
             partition_ids: List[int] = field(default_factory=list)
-            max_lag_seconds: Union[datetime.timedelta, None] = None
-            read_from: Union[int, float, datetime.datetime, None] = None
+            max_lag: Optional[datetime.timedelta] = None
+            read_from: Optional[datetime.datetime] = None
 
             def to_proto(
                 self,
             ) -> ydb_topic_pb2.StreamReadMessage.InitRequest.TopicReadSettings:
                 res = ydb_topic_pb2.StreamReadMessage.InitRequest.TopicReadSettings()
                 res.path = self.path
                 res.partition_ids.extend(self.partition_ids)
-                if self.max_lag_seconds is not None:
-                    res.max_lag = proto_duration_from_timedelta(self.max_lag_seconds)
+                max_lag = proto_duration_from_timedelta(self.max_lag)
+                if max_lag is not None:
```

Comment:
It may be no need technically, but need by [TypeHints](https://github.com/ydb-platform/ydb-python-sdk/blob/b839bd37fabb3c21fe6c9a87ee9911c4771b3822/ydb/_grpc/v4/protos/ydb_topic_pb2.pyi#L490)